### PR TITLE
Ws bugfix

### PR DESCRIPTION
**Pull request by OnyxPay**

### CODE REVIEW

File: src/pages/requests/ClosedRequests.js [hunk 1]
```diff
@@ -64,15 +64,15 @@ class ClosedRequests extends Component {
 				params.type = requestType;
 				params.status = "rejected,completed";
 				params.user = "maker";
-				getOpRequests({ params, requestType, fetchActive: false, isInitiator: true });
+				getOpMessages({ params, requestType, fetchActive: false, isInitiator: true });
```
Comment (goonerDp) [inline src/pages/requests/ClosedRequests.js]: Closed requests for initiator should be **fetched from different endpoint!** Now they are empty.

File: src/redux/messages.js [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+	operationMessageStatusNames,
+	requestStatusNames,
+	requestStatus,
+} from "api/constants";
+import { initState, enumerateItems, wsEventTypeToStatus } from "./requestsCommon";
+
+export const GET_OPERATION_MESSAGES_SUCCESS = "GET_OPERATION_MESSAGES_SUCCESS";
+export const GET_OPERATION_MESSAGES_FAILURE = "GET_OPERATION_MESSAGES_FAILURE";
+export const GET_OPERATION_MESSAGES = "GET_OPERATION_MESSAGES";
+export const DISABLE_OPERATION_REQ = "DISABLE_OPERATION_REQ";
```
Comment (goonerDp) [inline src/redux/messages.js]: DISABLE_OPERATION_REQ  can be imported from src\redux\requests.js

File: src/redux/messages.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+			payload: data,
+			fetchActive: fetchActive,
+		});
+	} catch (e) {
+		console.log(e);
+		dispatch({ type: GET_OPERATION_MESSAGES_FAILURE });
+	}
+};
+
+export const disableRequest = requestId => {
```
Comment (goonerDp) [inline src/redux/messages.js]: can be imported from src\redux\requests.js

File: src/redux/messages.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+	}
+	return enumerateItems(state, pred);
+};
+
+export const getOpMessages = ({
+	params = {}, // get params
+	requestType, // deposit | withdraw | buy_onyx_cash
+	fetchActive, // true | false
+}) => async dispatch => {
+	dispatch({ type: GET_OPERATION_REQUESTS });
```
Comment (goonerDp) [inline src/redux/messages.js]: Should be GET_OPERATION_MESSAGES_REQUEST

File: src/pages/requests/ClosedRequests.js (hunk 1 above)
Comment (OnyxPay) [reply src/pages/requests/ClosedRequests.js]: fixed

File: src/redux/messages.js (hunk 2 above)
Comment (OnyxPay) [reply src/redux/messages.js]: done

File: src/redux/messages.js (hunk 3 above)
Comment (OnyxPay) [reply src/redux/messages.js]: I removed it here

File: src/redux/messages.js (hunk 4 above)
Comment (OnyxPay) [reply src/redux/messages.js]: +

**Pull request merged**